src/core/client/admin/routes/Moderate/Queue/ModerationQueue.tsx

```diff
@@ -0,0 +1,20 @@
+import { useRouter } from "found";
+import { FunctionComponent, useEffect } from "react";
+
+interface Props {
+  mode: "PRE" | "POST" | "%future added value" | null;
+}
+
+const ModerationQueue: FunctionComponent<Props> = ({ mode }) => {
+  const { router } = useRouter();
+  useEffect(() => {
+    if (mode === "PRE") {
+      router.replace("/admin/moderate/pending");
+    } else if (mode === "POST") {
+      router.replace("/admin/moderate/reported");
+    }
```

Comment:
You need to handle when the `storyID`, `siteID`, or `commentID` is passed. Check the possible invocations in `src/core/client/framework/helpers/getModerationLink.ts#getModerationLink` and adjust routes in the router to use this component.